packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -63,14 +75,21 @@ export class Analyzer {
     for (const fileName of rootFileNames) {
       modules.push(this.analyzeFile(fileName as AbsolutePath));
     }
-    return new Package(modules);
+    return new Package({
+      modules,
+      tsConfig: this.commandLine,
+      packageJson: this.packageJson,
+    });
   }
 
   analyzeFile(fileName: AbsolutePath) {
     const sourceFile = this.program.getSourceFile(fileName)!;
+    const sourcePath = absoluteToPackage(fileName, this.packageRoot);
+    const jsPath = sourceToJs(sourcePath, this.packageRoot, this.commandLine);
```

Comment:
You can use `ts.getOutputFileNames`

Reply:
Done. Added some comments, but it looks like in order to use `getOutputFileNames`, you need to pass `configFileName` into `parseJsonConfigFileContent`, at which point you _can't_ pass `basePath` (afaict once you pass `configFileName` in, files automatically become relative to it, which I guess makes sense?):

```ts
    // Note that passing `packageRoot` for `basePath` works, but
    // `getOutputFileNames` will fail without passing `configFileName`; once you
    // pass that, it looks like paths are relative to the `configFileName`
    // location (which is also under `packageRoot`), in which case `basePath`
    // shouldn't duplicate `packageRoot`
    this.commandLine = ts.parseJsonConfigFileContent(
      configFile.config /* json */,
      ts.sys /* host */,
      './' /* basePath */,
      undefined /* existingOptions */,
      configFileName /* configFileName */
    );
```